crates/swc_ecma_minifier/tests/fixture/next/feedback-2/codemirror/output.js

```diff
@@ -2365,8 +2365,8 @@
             return markedSpans ? markedSpans[n] : null;
         }
         function update(line, text, spans) {
-            var line1, text1, markedSpans, estimateHeight1, estHeight;
```

Comment:
Not sure if inlining of `markedSpans` is correct.

Reply:
`detachMarkedSpans(line1)` can change `markedSpans` or `spans`, so we need infection analysis

Reply:
I didn't checked rust diffs yet, so I'll postpone this

Reply:
I have actually think about this, used as param won't change its inline-ability -- If it's a primitive value, function cannot change it; if it's not, assign cannot preserve its original value, unless tuple/struct has arrived.

Reply:
I'm not talking about the case where a variable is used as a parameter.
I mean the function call of `detachMarkedSpans` can change variables

Reply:
That's why I implemented inliner mainly by sequential inliner

Reply:
closure compiler has done this, see [here](https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520SIMPLE_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%2540formatting%2520pretty_print%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%252F%252F%2520ADD%2520YOUR%2520CODE%2520HERE%250Aexport%2520function%2520update(line%252C%2520text%252C%2520spans)%2520%257B%250A%2520%2520%2520%2520var%2520line1%252C%2520text1%252C%2520markedSpans%252C%2520estimateHeight1%252C%2520estHeight%253B%250A%2520%2520%2520%2520line1%2520%253D%2520line%252C%2520text1%2520%253D%2520text%252C%2520markedSpans%2520%253D%2520spans%252C%2520estimateHeight1%2520%253D%2520estimateHeight%252C%2520line1.text%2520%253D%2520text1%252C%2520line1.stateAfter%2520%2526%2526%2520(line1.stateAfter%2520%253D%2520null)%252C%2520line1.styles%2520%2526%2526%2520(line1.styles%2520%253D%2520null)%252C%2520null%2520!%253D%2520line1.order%2520%2526%2526%2520(line1.order%2520%253D%2520null)%252C%2520detachMarkedSpans(line1)%252C%2520attachMarkedSpans(line1%252C%2520markedSpans)%252C%2520(estHeight%2520%253D%2520estimateHeight1%2520%253F%2520estimateHeight1(line1)%2520%253A%25201)%2520!%253D%2520line1.height%2520%2526%2526%2520updateLineHeight(line1%252C%2520estHeight)%252C%2520signalLater(line%252C%2520%2522change%2522%252C%2520line%252C%2520change)%253B%250A%257D%250A%250A). Although the correctness of gcc is very quesionable.

Reply:
Github seems have to be bugged. Used as a function parameter won't make it un inlineable -- if it's a primitive value, function won't be able to change it; if it's an object, assign won't be able to copy it. So it's safe to inline it anyway

Reply:
I checked it using example code which provides getter/setters with side-effect and found that this is fine